marimo/_server/api/interrupt.py

```diff
@@ -55,18 +55,28 @@ def _interrupt_handler(self) -> None:
             self.shutdown()
 
         try:
-            if GLOBAL_SETTINGS.YES:
-                self.shutdown()
-                return
+            try:
+                if GLOBAL_SETTINGS.YES:
+                    self.shutdown()
+                    return
 
-            response = input(
-                f"\r{TAB}\033[1;32mAre you sure you want to quit?\033[0m "
-                "\033[1m(y/n)\033[0m: "
-            )
-            if response.lower().strip() == "y":
+                response = input(
+                    f"\r{TAB}Are you sure you want to quit? (y/n): "
```

Comment:
I think we have the colors.py lib if we want to add back parity

Reply:
For some reason `input` is not colorizing and instead showing the ANSI codes. I see this on my machine and another user saw it too (Discord), so simpler to just not colorize this.